tests/Integration/Behaviour/Features/Context/Domain/AliasFeatureContext.php

```diff
@@ -132,6 +158,14 @@ private function assertAliasProperties(array $expectedData, array $aliases): arr
                 'Unexpected alias reference'
             );
 
+            if (!empty($expectedAlias['active'])) {
```

Comment:
I don't think we have a case where "active" would be asserted against empty string or 0. When it is set in the table its either true or false. If you don't want to assert it you don't provide it in table at all.
```suggestion
            if (isset($expectedAlias['active'])) {
```